### DISCUSSION TITLE
What does photoprism doing when scanning a very large video file?

### DISCUSSION THREAD

**Discussion by ChowDPa02k** [photoprism/photoprism#4375] [Q&A] [answered]
Everytime when the indexing meet video files over 1GB, it will stuck for a while then show up a warning log "file size too big", then nothing was changed.
What did photoprism do at that time?
There are several of oversized video files in my library, making an index job costs about 3600s, and recently i transcoded them and remove the original videos, the indexing job fasten up to about 500s.

**Comment by lastzero** [comment, marked as answer]
Note that you can change the file size limit with `PHOTOPRISM_ORIGINALS_LIMIT` as described in our documentation:
- https://docs.photoprism.app/getting-started/config-options/#storage

Without knowing more about these files and your runtime environment, e.g. which file system (local, remote) you use, I can only guess what exactly is happening. For example, the time might be needed to calculate the checksum of the file to determine if the file has already been indexed. Our software is open source, so you can have a look at the source code to see what it does.

**Comment by RonnieBlaze** [comment]
I added that to my options.yml file and restarted and it did not work. is there another place to put it?

EDIT:
fixed it by adding OriginalsLimit instead of PHOTOPRISM_ORIGINALS_LIMIT to my options.yml

**Reply by DJKeterson** [reply]
PHOTOPRISM_ORIGINALS_LIMIT would be done in your compose file instead.
